src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/service/ApplicationService.kt

```diff
@@ -792,6 +819,7 @@ class ApplicationService(
       apType = when {
```

Comment:
i may be missing it, but are we using SubmitApprovedPremisesApplication.apType if defined like we do in the update function?

Reply:
This was an oversight I think, I've changed this.